henriquedsg89/pacman
# Add Schema api service

This PR add an end-point to request XML schema validation and to list the validations that has been done by PacMan.

What is included in this PR:
- Add spray.io (http://spray.io)
- Add Feed Validation REST Service
- Add JSON serializers to Feed Validation model class
- Add Feed Validator Actor
- Add XML Downloader Actor
- Add StreamSource Unmarshaller
- Add Schema Validator
- Add Schema Validator Error Handler
- Add GXML Schema Validator
- Add Feed Validation model class
- Add Feed Validation Request model class
- Add trait FeedValidationDBService
- Add class InMemoryFeedValidationDBService extends FeedValidationDBService
- Add logback (http://logback.qos.ch/)

ps.: There is a lot of changes in this PR, i should never do this again.
